### ISSUE TITLE
[FEATURE] make releases with the plugins

### ISSUE THREAD

**Issue by ghost** [mov-cli/mov-cli#304]
**Describe the feature / change you'd like.**
The plugins have no releases or no recent functioning releases. That's ALL of them, this is why i report this here.

**Additional context:**
A release signals witch version users should be using.

**Comment by THEGOLDENPRO** [comment]
Yeah sorry about that. I assume you're using releases for packaging. I'm usually late with releases but soon I'll update them and keep them up to date.

@r3tr0ananas also when you release a pypi release for plugins remember to also do a GitHub one.

You can leave the mov-cli GitHub releases to me.

**Comment by ananasmoe** [comment]
> @r3tr0ananas also when you release a pypi release for plugins remember to also do a GitHub one.

![1232865690059542629](https://github.com/mov-cli/mov-cli/assets/132799819/ac2ae185-00e0-405e-8df3-4b9cf4383879)

**Comment by ghost** [comment]
I'm not packaging for a distro. It's for the mpr (like the AUR but for deb packages).

https://mpr.makedeb.org/packages/mov-cli

**Comment by THEGOLDENPRO** [comment]
A github release has been pushed everything. I don't know how the debian packages work. Typically on arch, python packages are packaged in a way it just grabs the source files from the [github releases page](https://github.com/mov-cli/mov-cli/releases).

So as long as that is updated, releases will pull in. Idk what is the case with the debian packages as we aren't the one's packaging them.

**Comment by ghost** [comment]
I mean, i'm not part of the official distro, i'm just a user making packages for the equivalent of a AUR for debian.

**Comment by ananasmoe** [comment]
> I mean, i'm not part of the official distro, i'm just a user making packages for the equivalent of a AUR for debian.

So you are making the packages on the [mpr](https://mpr.makedeb.org/), as I understand correctly?

**Comment by ghost** [comment]
This is a big cultural difference with windows. I'm assuming you are windows guys.

a Linux distro has basically an official "app store" called repository, with open source software packaged by the maintainers of the distro it self. This doesn't exist in windows. The equivalent would be for microsoft it self to package a bunch of software for windows. From python, gcc and the kernel  to chromium and firefox.... And all that is managed by a package manager keeping track of what's installed, the dependencies and keep stuff compatible, that too doesn't exist in windows. In linux you can upgrade everything with one command, from the kernel to soduku. Instead of every software having it's own way to upgrade it self. It's a disgrace that microsoft didn't copied this system.

Additional to the official "appstore" there are also user generated packages/repositories by random people on the internet. I'm just a random person on the internet that packaged mov-cli.

Long story short, package management in linux is extremely good, and linux users are totally addicted to it. And this is why i rather package mov-cli rather then use pip.

**Comment by ananasmoe** [comment]
> This is a big cultural difference with windows. I'm assuming you are windows guys.
> 
> a Linux distro has basically an official "app store" called repository, with open source software packaged by the maintainers of the distro it self. This doesn't exist in windows. The equivalent would be for microsoft it self to package a bunch of software for windows. From python, gcc and the kernel to chromium and firefox.... And all that is managed by a package manager keeping track of what's installed, the dependencies and keep stuff compatible, that too doesn't exist in windows. In linux you can upgrade everything with one command, from the kernel to soduku. Instead of every software having it's own way to upgrade it self. It's a disgrace that microsoft didn't copied this system.
> 
> Additional to the official "appstore" there are also user generated packages/repositories by random people on the internet. I'm just a random person on the internet that packaged mov-cli.
> 
> Long story short, package management in linux is extremely good, and linux users are totally addicted to it. And this is why i rather package mov-cli rather then use pip.

![image](https://github.com/mov-cli/mov-cli/assets/132799819/d44cb71d-d93f-4d6a-b9eb-0a224734d152)

i use arch btw :3

**Comment by THEGOLDENPRO** [comment]
> This is a big cultural difference with windows. I'm assuming you are windows guys.
> 
> a Linux distro has basically an official "app store" called repository, with open source software packaged by the maintainers of the distro it self. This doesn't exist in windows. The equivalent would be for microsoft it self to package a bunch of software for windows. From python, gcc and the kernel  to chromium and firefox.... And all that is managed by a package manager keeping track of what's installed, the dependencies and keep stuff compatible, that too doesn't exist in windows. In linux you can upgrade everything with one command, from the kernel to soduku. Instead of every software having it's own way to upgrade it self. It's a disgrace that microsoft didn't copied this system.
> 
> Additional to the official "appstore" there are also user generated packages/repositories by random people on the internet. I'm just a random person on the internet that packaged mov-cli.
> 
> Long story short, package management in linux is extremely good, and linux users are totally addicted to it. And this is why i rather package mov-cli rather then use pip.

Yeah we are aware of this. I am actually working on an Arch package as we speak. We just currently recommend pipx to users as we know it 100% works and is available to all distros. More official distro packages will make way in the future. 👍

**Comment by ghost** [comment]
lol... then... mpr is like the AUR.
